src/ui/AddPhotoBlock/AddPhotoBlock.jsx

```diff
@@ -0,0 +1,39 @@
+import React, { useState } from 'react';
+import './AddPhotoBlock.scss';
+
+const AddPhotoBlock = ({ photo, setPhoto, name, sizeLimit, labelText }) => {
+  const [errorText, setErrorText] = useState('');
+
+  const handlePhoto = (e) => {
+    const [loadingFile] = e.target.files;
+    if (loadingFile.size > sizeLimit) {
+      setErrorText('Изображение больше 5 МБ');
+      setPhoto('');
+    } else if (!/(jpg)|(jpeg)|(png)|(bmp)/.test(loadingFile.type)) {
+      setErrorText('Неверный тип файла');
+      setPhoto('');
+    } else {
```

Comment:
```suggestion
 const allowedTypes = ['image/jpeg', 'image/png', 'image/bmp'];

    if (!allowedTypes.includes(loadingFile.type)) {
      setErrorText('Неверный тип файла');
      setPhoto('');
      return;
    }
  
```
легче читается, если вынести типы в константу